blaze822.c

```diff
@@ -152,14 +152,18 @@ skip_comment(char *s)
 // never writes more than dstmax to dst
 // returns how many bytes were appended
 static size_t
-safe_append(char *dst, size_t dstmax, char *strbeg, char *strend)
+safe_append(char *dst, size_t dstmax, char *postbeg, char *postend)
```

Comment:
Sorry for the rename, but I wanted `xxxlen`, and `strlen` is not a cool variable name.